next-cloudinary/src/components/UploadWidget/UploadButton.js

```diff
@@ -0,0 +1,23 @@
+import { default as UploadWidget } from "./index";
+
+const UploadButton = ({ onUpload, options, signed, label, ...props }) => {
+  return (
+    <>
+      <UploadWidget signed={signed} options={options} onUpload={onUpload}>
+        {({ open }) => {
+          function handleOnClick(e) {
+            e.preventDefault();
+            open();
+          }
+          return (
+            <button onClick={handleOnClick} {...props}>
+              {label || "Upload an Image"}
```

Comment:
what do you think about using `children` for this instead of a `label` prop? my thinking is it gives people more flexibility for how they customize the button